src/screens/Hooks/OMPSwitchHooks.res

```diff
@@ -158,9 +157,33 @@ let useInternalSwitch = () => {
     } catch {
     | Exn.Error(e) => {
         let err = Exn.message(e)->Option.getOr("Failed to switch!")
-        showToast(~message=err, ~toastType=ToastError)
         Exn.raiseError(err)
       }
     }
   }
 }
+
+let useOMPData = () => {
+  open OMPSwitchUtils
+  let merchantList = Recoil.useRecoilValueFromAtom(HyperswitchAtom.merchantListAtom)
+  let orgList = Recoil.useRecoilValueFromAtom(HyperswitchAtom.orgListAtom)
+  let profileList = Recoil.useRecoilValueFromAtom(HyperswitchAtom.profileListAtom)
+  let {userInfo} = React.useContext(UserInfoProvider.defaultContext)
+  let getList: unit => OMPSwitchTypes.ompList = () => {
+    {
+      orgList,
+      merchantList,
+      profileList,
+    }
+  }
```

Comment:
```suggestion
  let getList: unit => OMPSwitchTypes.ompList = () => {
      orgList,
      merchantList,
      profileList,
  }
```

Reply:
This represents a getter function for all the list